app/views/games/show.html.erb

```diff
@@ -1,3 +1,31 @@
+<script type="text/javascript">
+  $(function() {
+    $('.piece').draggable({
+      cursor: 'crosshair'
+    });
+
+    $('.left').droppable({
+      drop: function(event, ui) {
+          alert("X position = " + ui.draggable.data('x') + "  Y position = " + ui.draggable.data('y'))
+          $.ajax({
+            type: 'PUT',
+            url: ui.draggable.data('update-game'),
```

Comment:
do you mean the updated url should be pieces id. ive tried to implement that but with no luck. it makes sense to me logically but maybe some bugs to fix up. or routing issues because there is no piece path, just game_pieces_path